docker-compose.yml

```diff
@@ -19,5 +21,12 @@ services:
     env_file:
       - ./app/.env
 
+  db-test:
```

Comment:
why you need a separate test db?

Reply:
this is for creating new account and etc and drop test data